From eb9cc56b34254a1d72a130e6b7a01a46d271f1b0 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Sat, 21 Oct 2017 10:33:06 +0200
Subject: [PATCH] build: don't override CC/CFLAGS/LDFLAGS

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 5d4b38b..488fcaf 100644
--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,12 @@
 # C compiler to use; clang works as well
-CC=gcc
+CC?=gcc
 
 # Flags to pass to the C compiler; these are simply the default
 # Arch Linux C compiler flags
-CFLAGS= -O2 -fstack-protector-strong
+CFLAGS?= -O2 -fstack-protector-strong
 
 # Flags to pass to the linker; required to link against libbsd
-LDFLAGS= -lbsd
+LDFLAGS+= -lbsd
 
 all:
 	$(CC) brightlight.c -o brightlight $(CFLAGS) $(LDFLAGS)
