From 164d57aeb1ee48d2222f15982269bf63ac3b2b63 Mon Sep 17 00:00:00 2001
From: "Justin W. Flory" <git@jwf.io>
Date: Mon, 29 Apr 2019 21:00:13 -0400
Subject: [PATCH 1/1] Makefile: Change compile options for Fedora RPM package

---
 Makefile | 15 +++++++++------
 dinit.c  |  2 +-
 2 files changed, 10 insertions(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 88305ce..1e9da7a 100644
--- a/Makefile
+++ b/Makefile
@@ -1,13 +1,16 @@
 # Makefile for dungeon
 
 # Where to install the program
-BINDIR = /usr/games
+# - overridden for RPM package (set in environment variable)
+# BINDIR = /usr/games
 
 # Where to install the data file
-DATADIR = /usr/games/lib
+# - overridden for RPM package (set in environment variable)
+# DATADIR = /usr/games/lib
 
 # Where to install the man page
-MANDIR = /usr/share/man
+# - overridden for RPM package (set in environment variable)
+# MANDIR = /usr/share/man
 
 # The dungeon program provides a ``more'' facility which tries to
 # figure out how many rows the terminal has.  Several mechanisms are
@@ -42,7 +45,7 @@ TERMFLAG =
 # Uncomment the following line if you want to have access to the game
 # debugging tool.  This is invoked by typing "gdt".  It is not much
 # use except for debugging.
-GDTFLAG = -DALLOW_GDT
+# GDTFLAG = -DALLOW_GDT
 
 # Compilation flags
 CFLAGS = -g #-static
@@ -69,7 +72,7 @@ dungeon: $(OBJS) dtextc.dat
 	$(CC) $(CFLAGS) -o zork $(OBJS) $(LIBS)
 
 install: zork dtextc.dat
-	mkdir -p $(BINDIR) $(LIBDIR) $(MANDIR)/man6
+	mkdir -p $(BINDIR) $(DATADIR) $(LIBDIR) $(MANDIR)/man6
 	cp zork $(BINDIR)
 	cp dtextc.dat $(DATADIR)
 	cp dungeon.6 $(MANDIR)/man6/
@@ -93,7 +96,7 @@ local.o: local.c funcs.h vars.h
 	$(CC) $(CFLAGS) $(GDTFLAG) -c local.c
 
 supp.o: supp.c funcs.h vars.h
-	$(CC) $(CFLAGS) $(TERMFLAG) -c supp.c	
+	$(CC) $(CFLAGS) $(TERMFLAG) -c supp.c
 
 actors.o: funcs.h vars.h
 ballop.o: funcs.h vars.h
diff --git a/dinit.c b/dinit.c
index d687cf4..cda5878 100644
--- a/dinit.c
+++ b/dinit.c
@@ -24,7 +24,7 @@ FILE *dbfile;
 #define TEXTFILE "lib:dtextc.dat"
 #else /* ! __AMOS__ */
 #ifdef unix
-#define TEXTFILE "/usr/games/lib/dunlib/dtextc.dat"
+#define TEXTFILE "/usr/share/zork/dtextc.dat"
 #else /* ! unix */
  I need a definition for TEXTFILE
 #endif /* ! unix */
-- 
2.20.1

